addons/mail/static/src/components/message/message.xml

```diff
@@ -157,7 +158,17 @@
                         </t>
                     </t>
                     <div class="o_Message_content" t-ref="content">
-                        <div class="o_Message_prettyBody" t-ref="prettyBody"/><!-- message.prettyBody is inserted here from _update() -->
+                        <t t-if="message.hasSendError">
+                            <span class="not-delivred">
+                                <i class="fa fa-exclamation-circle" />
+                                This message was not delivered
+                            </span>
+                            <br />
+                            <a href="" class="o_Message_sendAgain not-delivred" t-on-click="_onClickSendAgain" title="Try again">
```

Comment:
Considering the link has Retry as text, "try again" title does not seem to have any value.